lib/process-services-cloud/src/lib/form/components/widgets/attach-file/attach-file-cloud-widget.component.spec.ts

```diff
@@ -376,32 +318,6 @@ describe('AttachFileCloudWidgetComponent', () => {
             expect(openUploadFileDialogSpy).toHaveBeenCalledWith('-my-', 'multiple', true, true);
         });
 
-        it('Should set default user alias (-my-) as rootNodeId if destinationFolderPath is not defined', async () => {
-            const getAliasAndPathSpy = spyOn(widget, 'getAliasAndRelativePathFromDestinationFolderPath').and.callThrough();
-            createUploadWidgetField(new FormModel(), 'attach-file-alfresco', [], allSourceWithoutDestinationFolderPath);
-            fixture.detectChanges();
-            await fixture.whenStable();
-            clickOnAttachFileWidget('attach-file-alfresco');
-            fixture.detectChanges();
-            await fixture.whenStable();
-
-            expect(getAliasAndPathSpy).not.toHaveBeenCalled();
-            expect(widget.rootNodeId).toEqual('-my-');
-        });
-
-        it('Should set default user alias (-my-) as rootNodeId if value property missing from destinationFolderPath', async () => {
-            const getAliasAndPathSpy = spyOn(widget, 'getAliasAndRelativePathFromDestinationFolderPath').and.callThrough();
-            createUploadWidgetField(new FormModel(), 'attach-file-alfresco', [], allSourceWithoutValueProperty);
-            fixture.detectChanges();
-            await fixture.whenStable();
-            clickOnAttachFileWidget('attach-file-alfresco');
-            fixture.detectChanges();
-            await fixture.whenStable();
-
-            expect(getAliasAndPathSpy).not.toHaveBeenCalled();
-            expect(widget.rootNodeId).toEqual('-my-');
-        });
```

Comment:
Are you sure you have covered every single case that you removed from here? Because I see you have changed the tests not just move them somewhere else

Reply:
yup, removed tests were testing default cases. Now I moved default cases to service level so I have covered them as part service unit tests